dependencies/vaticle/repositories.bzl

```diff
@@ -42,7 +42,7 @@ def vaticle_typedb_protocol():
     git_repository(
         name = "vaticle_typedb_protocol",
         remote = "https://github.com/vaticle/typedb-protocol",
-        tag = "2.1.2", # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_typedb_protocol
+        commit = "008f0fc3bb11c221214a9b47899168680cf111be", # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_typedb_protocol
```

Comment:
you wanna bump all the deps to the latest? some of them were reverted to last released for the 2.2.0 release

Reply:
Done in https://github.com/vaticle/typedb/pull/6386/commits/7eeb2e3a030670453cae10621114c281913a7959